src/AzureClient/Visualization/HistogramEncoders.cs

```diff
@@ -20,9 +20,35 @@ internal class Histogram : Dictionary<string, double>
 
     internal static class HistogramExtensions
     {
-        internal static Histogram ToHistogram(this Stream stream, ILogger? logger = null)
+        internal static Histogram ToHistogram(this Stream stream, ILogger? logger = null, bool isSimulatorOutput = false)
         {
             var output = new StreamReader(stream).ReadToEnd();
+            if (isSimulatorOutput)
+            {
+                // This routine seems to be using what I think may be an older format
+                // (the az quantum cli had handling for both):  {"Histogram":["0",0.5,"1",0.5]}
+                output = "{ \"Histogram\" : [ \"" + output.Trim() + "\", 1.0 ] }";
+
+                // TODO: Make this more general. The corresponding implementation from the az quantum cli is:
```

Comment:
This code is still a work progress and will be addressed in an upcoming PR that will update the output format of jobs submitted to the full state simulator. Until then this comment out section should remain.

Reply:
I've added a link to https://github.com/microsoft/iqsharp/issues/607.